src/descriptor/bare.rs

```diff
@@ -359,7 +349,9 @@ impl<Pk: MiniscriptKey> DescriptorTrait<Pk> for Pkh<Pk> {
     where
         Pk: ToPublicKey,
     {
-        self.spk()
+        // TODO: This is broken, remove the hard coded network.
```

Comment:
In a56fe857b0647318dacf6e386895ec9ca0fd6082:

I don't understand this comment. The network was hardcoded i the old code, and anyway it doesn't affect the output of this function at all.

Reply:
Oh, you are correct, I am totally wrong with this code comment. It should perhaps be:
```
        // Fine to hard code the address here because we immediately call
        // `script_pubkey` which does not use the `network` field of `Address`.
```